assets/stylesheets/custom_messages.css

```diff
@@ -36,6 +36,11 @@
   color: white;
 }
 
-p.toggle-enable {
+div.message-customize-menu {
   text-align: right;
+}
+div.message-customize-menu p {
+  color: gray;
+  font-size: 10px;
+  margin: 5px;
```

Comment:
I think `margin: 5px 0px;` is better.

Reply:
`margin: 5px` is margin-left and margint-right were also 5px.
I fixed to `margin: 5px 0px;`.

Reply:
I agree.